template/example/README.md

```diff
@@ -1,3 +1,5 @@
 # This is a terraform module example
+The current directory contains terraform related files that use the module in `../`.
```

Comment:
```suggestion
# This is a terraform module example
The current directory contains terraform related files that use the module in `../`. See [test_example](../test_example/example/) for an example on how to do this.
```

Reply:
maybe just ?
```suggestion
# This is a terraform module example
The current directory contains terraform related files that use the module in `../`. See [test_example](../test_example/example/).
```

Reply:
Yeah, that's better! 

I like words if you can't tell :grinning: